mm2src/mm2_main/tests/mm2_tests/mm2_tests_inner.rs

```diff
@@ -256,6 +256,80 @@ fn test_my_balance() {
     assert_eq!(my_address, "RRnMcSeKiLrNdbp91qNVQwwXx5azD4S4CD");
 }
 
+#[test]
```

Comment:
Maybe good to add a test to spend p2pk

Reply:
Yeah I actually should do.
I remember I didn't because the address will run out of testnet coins eventually, but such a test should fit in the docker tests :+1:

Reply:
Just got my head back. It's not really possible to test these in the docker tests because the native client doesn't support p2pk (only electrum for now).
Docker tests are for native only (as of now).
I will leave a todo here to not forget to add such a test when we have electrum nodes in docker tests.